python/ray/autoscaler/gcp/example-tpu-pod.yaml

```diff
@@ -0,0 +1,49 @@
+# A unique identifier for the head node and workers of this cluster.
+cluster_name: tpupodtest
+
+max_workers: 2
+
+available_node_types:
+    ray_head_default:
+        min_workers: 0
+        max_workers: 0
+        resources: {"CPU": 0}
+        # Provider-specific config for this node type, e.g. instance type. By default
+        # Ray will auto-configure unspecified fields such as subnets and ssh-keys.
+        # For more documentation on available fields, see:
+        # https://cloud.google.com/compute/docs/reference/rest/v1/instances/insert
+        node_config:
+            machineType: n1-standard-4
+            disks:
+              - boot: true
+                autoDelete: true
+                type: PERSISTENT
+                initializeParams:
+                  diskSizeGb: 50
+                  # See https://cloud.google.com/compute/docs/images for more images
+                  sourceImage: projects/ubuntu-os-cloud/global/images/family/ubuntu-2004-lts
+    ray_tpu:
+        min_workers: 1
+        resources: {"TPU": 1}  # use TPU custom resource in your code
+        node_config:
+            acceleratorType: v4-16
```

Comment:
Should we add a comment to emphasize that v4-16 has two hosts?

Reply:
Done